gravitee-apim-portal-webui-next/src/app/registration/registration.component.scss

```
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use '../../scss/theme';
+
+:host {
+  display: flex;
+  justify-content: center;
+}
+
+.registration {
+  display: flex;
+  flex-flow: column;
+  gap: 20px;
```

We could reuse common styles here like in Login component. This will handle global paddings, centering and content width.


```suggestion
@use '../../scss/theme';
@use '../../scss/layout' as layout;

:host {
  @include layout.small-container-legacy();
}

.registration {
  display: flex;
  flex-flow: column;
  width: 100%;
  gap: 20px;
```

We also need to add `content` class on the form for this to work correctly:
https://github.com/gravitee-io/gravitee-api-management/pull/14707/changes#r2640323437